hknweb/academics/models/icsr.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+from .base_models import AcademicEntity
+
+
+class ICSR(AcademicEntity):
+    """
+    InstructorCourseSemesterRelation (ICSR)
+    """
+    # reference attributes
+    icsr_course = models.ForeignKey('Course', on_delete=models.CASCADE, related_name='icsr_course')
+    icsr_department = models.ForeignKey('Department', on_delete=models.CASCADE, related_name='icsr_department')
+    icsr_instructor = models.ForeignKey('Instructor', on_delete=models.CASCADE, related_name='icsr_instructor')
+    icsr_semester = models.ForeignKey('Semester', on_delete=models.CASCADE, related_name='icsr_semester')

```

Comment:
Again, don't see the need for the `icsr_` prefixing.

Reply:
See the previous comment